Libraries/Text/RCTTextAttributes.m

```diff
@@ -214,6 +216,22 @@ - (UIColor *)effectiveBackgroundColor
   return effectiveBackgroundColor ?: [UIColor clearColor];
 }
 
+- (NSString*)applyTextAttributesToText:(NSString*)text
```

Comment:
Space between type and pointer (NSString *) for both

Sorry for style nits, need prettier for objc!

Reply:
In the last round of feedback I was asked to fix the formatting to what it is now, which matches the formatting in the surrounding code (no spaces).   My personal preference is spaces, but I think my additions should match the surrounding code.

Reply:
There should be a space between NSString and *, the rest is fine

Reply:
gotcha; fixed.